.circleci/scripts/auto-rebase.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# init git
+git config user.email "circleci@bolt.com"
+git config user.name "Circle CI"
+
+baseBranch="master"
+# add the branches that you want to auto-rebase
+# use space to sparate them (don't use comma)
+autoRebaseBranches=("jz/rebase-custom1" "jz/rebase-custom2")
+
+for branchName in "${autoRebaseBranches[@]}"; do
+  echo "Auto-rebasing $branchName to $baseBranch..."
+  if ! (git checkout "$branchName"); then
+    echo "Failed to checkout branch $branchName"
+    exit 1
```

Comment:
Do we want to continue the process with other branches even if this one failed?

Reply:
I think we do because we may not block master for failing customization but we still want to know all branches that failed.

Reply:
fixed @usmanismail-bolt

Reply:
Shouldn't the exit 1 be return 1?

Reply:
@usmanismail-bolt good catch. It was actually fine because the function call is wrapped in parentheses, which would create a subshell, but for best practices I have changed to to **_return_**